src/requirements/data/majors/foodsci.ts

```diff
@@ -0,0 +1,276 @@
+      'Students are required to complete one concentration from Science, Safety and Business.\nWarning: For the optional electives in each concentration, we do not check whether the courses selected fulfil the minimum credits needed for that category.',
+    source: 'https://cals.cornell.edu/education/degrees-programs/food-science-major-minor',
+    fulfilledBy: 'toggleable',
+    // TODO: Can change Science and Business options to be fulfilled by credits instead, however would lose slot names.
+    fulfillmentOptions: {
+      Science: {
+        description:
+          'Required FDSC courses, and 3 >4000 FDSC electives, with at least 3 credits from group 1 and 2 credits from group 2.',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['FDSC 3950'],
+          ['FDSC 4100'],
+          ['FDSC 4190'],
+          ['FDSC 4230'],
+          ['FDSC 4010', 'FDSC 4040', 'FDSC 4050', 'FDSC 4220', 'FDSC 4250', 'FDSC 4400'],
+          [
+            'FDSC 4020',
+            'FDSC 4110',
+            'FDSC 4500',
+            'FDSC 4510',
+            'FDSC 4880',
+            'FDSC 5970',
+            'FDSC 6950',
+            'FDSC 4010',
+            'FDSC 4040',
+            'FDSC 4050',
+            'FDSC 4220',
+            'FDSC 4250',
+            'FDSC 4400',
+          ]
+        ),
+        perSlotMinCount: [1, 1, 1, 1, 1, 1],
+        slotNames: [
+          'FDSC 3950',
+          'FDSC 4100',
+          'FDSC 4190',
+          'FDSC 4230',
+          'Group 1 >4000 electives',
+          'Group 2 >4000 electives',
+        ],
+      },
+      Safety: {
+        description:
+          'Required FDSC courses, epidemiology course, risk analysis and management course, microbial pathogenesis course, and 2 credits of >4000 FDSC electives',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['FDSC 3950'],
+          ['FDSC 4230', 'FDSC 4250'],
+          ['FDSC 5970'],
+          ['NS 3600', 'PAM 3280', 'ENTOM 4250'],
+          ['PAM 4240', 'DSOC 4080', 'ORIE 2380'],
+          ['BIOMS 4040', 'BIOMS 4090', 'BIOMS 4150', 'BIOMS 4340', 'BIOMG 4390'],
+          [
+            'FDSC 4010',
+            'FDSC 4020',
+            'FDSC 4040',
+            'FDSC 4050',
+            'FDSC 4100',
+            'FDSC 4110',
+            'FDSC 4220',
+            'FDSC 4230',
+            'FDSC 4250',
+            'FDSC 4400',
+            'FDSC 4500',
+            'FDSC 4880',
+            'FDSC 5400',
+          ]
+        ),
+        perSlotMinCount: [1, 1, 1, 1, 1, 1, 1],
+        slotNames: [
+          'FDSC 3950',
+          'FDSC 4230 or FDSC 4250',
+          'FDSC 5970',
+          'NS 3600 or PAM 3280 or ENTOM 4250 (in preferred order)',
+          'PAM 4240 or DSOC 4080 or ORIE 2380 (in preferred order)',
+          'BIOMS 4040 or BIOMS 4090 or BIOMS 4150 or BIOMS 4340 or BIOMG 4390 (in preferred order)',
+          '2 credits of >4000 FDSC electives',
```

Comment:
Nit: To be consistent with other subrequirements that mention credit minimums, can you say 4000+ instead of >4000? This is for here and other places as well.